### ISSUE TITLE
Namco Museum hangs w/ black screen when NoAudioDelay is true in mupen64plus.conf

### ISSUE THREAD

**Issue by richard42** [mupen64plus/mupen64plus-user-issues#48]
Originally reported on Google Code with ID 48

```
- Gentoo linux
- Affects 32-bit and 64-bit builds
- SVN rev 186 and older


```

Reported by `richard42g` on 2008-04-13 17:14:26

**Comment by richard42** [comment]
```
This has most likely something to do with timing issues, and is not really an issue
as audio sync is sometimes required for compatibility. But it might be interesting
to
take a look at.
```

Reported by `sgorman07` on 2008-04-14 04:47:34

**Comment by richard42** [comment]
```
The NoAudioDelay core option was removed from Mupen64Plus in rev 496 to resolve this
issue.  The "readme.pdf" included with Mupen64 v0.5 states that this option (along
with NoCompiledJump) were not necessary for v0.5, so presumably this option can be
safely removed.
```

Reported by `richard42g` on 2008-06-03 03:47:16
- **Status changed**: `Fixed`
